dev-docs/bidders/smartadserver.md

```diff
@@ -9,6 +9,7 @@ gdpr_supported: true
 schain_supported: true
 tcf2_supported: true
 usp_supported: true
+eids_supported: true
```

Comment:
this should be `userIds` rather than `eids_supported` I believe

```suggestion
userIds: britepoolId, criteo, digitrust, id5Id, identityLink, liveIntentId, netId, parrableId, pubCommonId, unifiedId
```

Reply:
Good catch, my mistake